### ISSUE TITLE
NetworkPolicy blocks readiness probes

### ISSUE THREAD

**Issue by laurivosandi** [mongodb/mongodb-kubernetes-operator#1087]
Please document how are NetworkPolicies supposed to be set up in order to make Mongo instances work with the operator

**Comment by laurivosandi** [comment]
So far I've got this:

```
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: mongodb-operator
spec:
  podSelector:
    matchLabels:
      app: mongodb-svc
  policyTypes:
  - Ingress
  - Egress
  ingress:
    - from:
      - podSelector: {}
      ports:
      - port: 27017
  egress:
    - to:
      - podSelector:
          matchLabels:
            app: mongodb-svc
      ports:
      - port: 27017
```

**Comment by dan-mckean** [comment]
Hi, I'm Dan and I'm the Product Manager for MongoDB's support of Kubernetes.

I'm doing some work right now to try and identify how the Community Operator is being used. The Community Operator is something I inherited when I started at MongoDB, but it doesn't get as much attention from us as we'd like and we're trying to understand how it's used in order to establish its future. It will help us prioritize future issues and PRs raised by the community 🙂

Here's a super short survey (it's much easier for us to review all the feedback that way!): [https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link)

 If you'd rather email me instead: [dan.mckean@mongodb.com](mailto:dan.mckean@mongodb.com?subject=MongoDB%20Community%20Operator%20feedback)

Thank you in advance!
Dan
